### ISSUE TITLE
[Visual requirements  -Howdy-for-pms]: “Powered by Groove” link is not getting underlined\Colour or chevron.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#12]
**User Experience**
Some users with visual/cognitive disabilities might find content more/only readable in a certain font, text size, or especially with only certain colour combination.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://howdy.ai/
Browser: Edge
 
**Repro Steps**
Step 1: Open the URL: https://hcl-szz1925.howdy.ai/dashboard
Step 2: Navigate to "Dashboard" tab and select it.
Step 3: Navigate to "Howdy for Your PM" link and select it.
Step 4: "Howdy for Your PM" page gets opened.
Step 5: Navigate to “Powered by Groove” link 
Step 6: Observe the issue.

**Actual Result**
“Powered by Groove” link is not getting underlined\Colour or chevron.
 
**Expected Result**
  “Powered by Groove” link should be either underlined or followed by Chevron\Colour for easy recognition from normal text

**Mas Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={9f1f0271-42a1-45b1-be00-31473ba2e579}

 
[MAS1.4.1_Howdy pm.zip](https://github.com/howdyai/howdy-website/files/2973721/MAS1.4.1_Howdy.pm.zip)
![MAS1 4 1_Howdy PM](https://user-images.githubusercontent.com/48272005/54473360-ea50b800-47fc-11e9-9db0-b2579ba0ae13.png)

**Comment by benbrown** [comment]
This page has been deprecated and removed from the site.

**Comment by v-gasikk** [comment]
Now this page is not available. Closing this issue based on developer comment.
